rplugin/node/vim-package-info/utils.js

```diff
@@ -4,19 +4,36 @@ if (!("vimnpmcache" in global)) {
   // you might think that we do not have to have two diffent objects
   // but there might be a single project with both package.json and Cargo.toml
   global.vimnpmcache = {
-    "package.json": {},
-    "Cargo.toml": {},
-    "requirements.txt": {}
+    "javascript": {},
+    "rust": {},
+    "python:requirements": {},
+    "python:pyproject": {}
   };
 }
 
+function determineFileKind(filename) {
+  if (filename.match(/^.*?requirements.txt$/)) {
+    return "python:requirements";
+  }
+  if (filename.match(/^pyproject.toml$/)) {
+    return "python:pyproject";
+  }
+  if (filename.match(/^Cargo.toml$/)) {
+    return "rust";
+  }
+  if (filename.match(/^project.json$/)) {
```

Comment:
it is `package.json`
```suggestion
  if (filename.match(/^package.json$/)) {
```